src/vivarium_gates_child_iv_iron/components/observers.py

```diff
@@ -79,6 +80,15 @@ def child_growth_risk_factor_stratification_mapper(self, row: pd.Series) -> str:
         }[row.squeeze()]
 
 
+class DisabilityObserver(DisabilityObserver_):
+    def on_post_setup(self, event: Event) -> None:
+        for cause in self._cause_components:
+            if cause.has_disability:
```

Comment:
better to say `if cause.has_disability or cause.name == 'disease_model...':` in a single if block

Reply:
Will update.